src/spatial/collision.js

```diff
@@ -349,39 +341,31 @@ Crafty.c("Collision", {
         return this;
     },
 
-
-    /**@
-     * #._createCollisionHandler
-     * @comp Collision
-     * @sign public void .checkHits(String component, Object collisionData)
-     * @param component - The name of the component for which this handler
-     * checks for collisions.
-     * @param collisionData - Collision data object used to track collisions with
-     * the specified component.
+    /**
```

Comment:
I've change `/**@` to `/**` (removed the `@`) not `/*` because of IDE highlighting, as I can see, you documentation gets only `/**@`?

And I deleted `#._createCollisionHandler` and `@comp Collision` 'cause of its uselessness, also moved the describtion to the top of jsdoc.